xml/System/Math.xml

```diff
@@ -4131,11 +4336,11 @@ Remainder = (Math.Abs(dividend) - (Math.Abs(divisor) *
         <Parameter Name="y" Type="System.Double" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="x">To be added.</param>
-        <param name="y">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="x">The first of two single-precision floating-point numbers to compare.</param>
+        <param name="y">The second of two single-precision floating-point numbers to compare.</param>
+        <summary>Returns the smaller magnitude of two single-precision floating-point numbers.</summary>
+        <returns>Parameter <paramref name="x" /> or <paramref name="y" />, whichever has the smaller magnitude. If <paramref name="x" />, or <paramref name="y" />, or both <paramref name="x" /> and <paramref name="y" /> are equal to <see cref="F:System.Double.NaN" />, <see cref="F:System.Double.NaN" /> is returned.</returns>
+        <remarks></remarks>
```

Comment:
`Math.Min(double, double)` will return `-5` for `Min(4, -5)`.
`Math.MinMagnitude(double, double)` will return `4` for `MinMagnitude(4, -5)` because `4` has the smaller overall magnitude (`Abs(4) < Abs(-5)`)